mina-daemon/scripts/gcloud-impersonator.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# This script is used to impersonate gcloud in the mina-daemon container and store the downloaded blocks in the /blocks directory
+
+echo "gcloud authentication disabled by gcloud-impersonator.sh"
```

Comment:
No, It is just because gcloud binary is required, for the upload-to-gcloud flag. So I need to have also a fake script that can be executed to bypass the gcloud authentication